src/scss/constants.scss

```diff
@@ -248,3 +248,9 @@ $z-layers: (
   max-height: $max;
   max-width: $max;
 }
+
+@mixin background-color($color) {
+  background-color: $color;
+  --UserAvatar__status-indicator--border-style: solid;
+  --UserAvatar__status-indicator--border-color: #{$color};
+}
```

Comment:
This allows ancestors of `UserAvatar` to not only set their background color, but also provide color for the indicator border inside the `UserAvatar`

Reply:
We shouldn't have the background and user avatar tied up. Let's have them separate. Any reason to have the border-color same as the background?

Reply:
Yes https://github.com/sparkletown/sparkle/pull/1891#discussion_r679379948

Reply:
Continuing discussion about this matter in the comment linked above^